test/verify/check-users

```diff
@@ -164,17 +164,17 @@ class TestAccounts(MachineCase):
 
         # Password is weak, lets change it to another weak - this should still not accept
         b.wait_in_text("#accounts-create-dialog .pf-c-form__helper-text.pf-m-error", "Password quality check failed:")
-        b.wait_in_text("#accounts-create-dialog .pf-c-form__helper-text.pf-m-error", "Click Create again to use the password anyway.")
```

Comment:
Oh, can you please replace these with checks that the new button was shown? It is equivalent to that.